src/corehost/cli/fxr/fx_resolver.messages.cpp

```diff
@@ -13,14 +13,14 @@ void fx_resolver_t::display_incompatible_framework_error(
     const pal::string_t& higher,
     const fx_reference_t& lower)
 {
-    assert(lower.get_patch_roll_fwd() != nullptr);
-    assert(lower.get_roll_fwd_on_no_candidate_fx() != nullptr);
+    assert(lower.get_apply_patches() != nullptr);
+    assert(lower.get_roll_forward() != nullptr);
 
-    trace::error(_X("The specified framework '%s', version '%s', patch_roll_fwd=%d, roll_fwd_on_no_candidate_fx=%d cannot roll-forward to the previously referenced version '%s'."),
+    trace::error(_X("The specified framework '%s', version '%s', apply_patches=%d, roll_forward=%d cannot roll-forward to the previously referenced version '%s'."),
```

Comment:
It would be nice to trace `roll_forward=%d` this as a string (also below)
Remove apply_patches 😉 (In case you didn't notice I have an opinion)